rainforest-cli.go

```diff
@@ -13,7 +13,7 @@ import (
 
 const (
 	// Version of the app in SemVer
-	version = "2.17.0"
+	version = "2.18.0"
```

Comment:
This should be changed in a followup [changelog PR](https://github.com/rainforestapp/rainforest-cli#changelog-pr).